src/plotman/plotman.py

```diff
@@ -163,9 +164,10 @@ def main():
 
         # Status report
         if args.cmd == 'status':
-            result = "{0}\n\n{1}".format(
+            result = "{0}\n\n{1}\n\nUpdated at: {2}".format(
                 reporting.status_report(jobs, get_term_width()),
-                reporting.summary(jobs)
+                reporting.summary(jobs),
+                datetime.datetime.today().strftime("%x %X"),
```

Comment:
You had said `%c` looked good.

```suggestion
                datetime.datetime.today().strftime("%c"),
```